platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/TransactionPool.java

```diff
@@ -170,7 +171,9 @@ public TransactionPool(
         }
 
         /** If the startFreezeAfterMin is 0, we don't send freeze transaction */
-        if (freezeConfig != null && platform.getSelfId().equalsMain(0) && freezeConfig.getStartFreezeAfterMin() > 0) {
+        if (freezeConfig != null
+                && platform.getSelfId().equals(NodeId.FIRST_NODE_ID)
```

Comment:
I always prefer `Objects.equals(platform.getSelfId(), NodeId.FIRST_NODE_ID)` to eliminate NPE possibilities

Reply:
changed